static/schemas/obms/amt-obm-service.json

```diff
@@ -27,6 +28,12 @@
                 }
             },
             "required": ["service", "config"]
```

Comment:
Also, I think it could be a problem to require `host` and `password` in config for patch since password might not be known by the user if the obm was auto-generated.